src/Utils/FileDiffCleaner.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Website\Utils;
+
+use Nette\Utils\Strings;
+
+final class FileDiffCleaner
+{
+    public function clean(string $fileDiff): string
+    {
+        // https://regex101.com/r/sI6GVY/1/
+        $match = Strings::match($fileDiff, '#^.*?@@\n(?<content>.*?)$#Us');
+        $fileDiff = $match['content'] ?? '';
+
+        if (Strings::contains($fileDiff, 'No newline at end of file')) {
+            $fileDiff = Strings::substring($fileDiff, 0, -strlen('\ No newline at end of file') - 1);
```

Comment:
What about `Strings::before()` instead? Could be more readable than this construction

Reply:
`Strings::replace()` would be even better.
Could you handle it + add test case for this class?